### PR TITLE
Release 3.0.0 - three kings

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by Lucpen** [comment]
Hi guys, I want to have the release ASAP but I am having some issues with the download test. As all the other tests pass, I think this issue is likely related to the fact that the pull_request_target runs with the code from the PR target, what do you guys think? @fellen31 @jemten @ramprasadn

**Review by jemten — APPROVED** [review]
Ready for holiday season 🎄 👑

**Comment by fellen31** [comment]
> Hi guys, I want to have the release ASAP but I am having some issues with the download test. As all the other tests pass, I think this issue is likely related to the fact that the pull_request_target runs with the code from the PR target, what do you guys think? @fellen31 @jemten @ramprasadn

Downloading the pipeline seems works fine for me locally. Perhaps something with the action.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,61 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 3.0.0 - Three Kings [2024-11-13]
```
Comment (Lucpen) [inline CHANGELOG.md:6]:
```suggestion
## 3.0.0 - Three Kings [2024-11-18]
```

**Pull request merged**